go.mod

```diff
@@ -181,6 +181,8 @@ require (
 replace (
 	// containerd main
 	github.com/containerd/containerd => github.com/containerd/containerd v1.6.1-0.20220621192144-0fac756b6452
+	// Temporary PR before making a patch in go-logrotate: https://github.com/fahedouch/go-logrotate/pull/4
+	github.com/fahedouch/go-logrotate => github.com/fahedouch/go-logrotate v0.1.2-0.20220714132646-81de1d3f32be
```

Comment:
@fahedouch Can you create a new release of go-logrotate?
LGTM then

Reply:
Yes 👍